### ISSUE TITLE
Broken links (404) in cookbook - arduino yun quick setup

### ISSUE THREAD

**Issue by branb** [crossbario/crossbarwww#75]
First one is under the section named "Activate the SD card", the link "these instructions" in the last paragraph has a typo which leads to a 404.

It is:
http://crossbar.io/iotcookbook/Arduino-YUn-Serial-to-WAMP-Bridge/

But should be:
http://crossbar.io/iotcookbook/Arduino-Yun-Serial-to-WAMP-Bridge/

Note aside: I'd suggest to change the link so it's not linking anymore the current text but "serial-to-WAMP-bridge" instead, and maybe rephrase the last sentence as well.

Also, at the bottom of the page in the section named "Next", the link "serial-to-WAMP-bridge" in the second paragraph has a typo which leads to a 404.

It is:
http://crossbar.io/iotcookbook/Arduin-Yun-Serial-to-WAMP-Bridge/

But should be:
http://crossbar.io/iotcookbook/Arduino-Yun-Serial-to-WAMP-Bridge/

**Comment by goeddea** [comment]
Thanks for the corrections!
